### ISSUE TITLE
The autocomplete function doesn't work.

### ISSUE THREAD

**Issue by NanXiao** [intellij-rust/intellij-rust#1061]
I use _IntelliJ IDEA_ (community edition 2016.3) to develop in Rust, but after installing the Rust plugin, the code auto-complete function doesn't work:
![capture](https://cloud.githubusercontent.com/assets/2338189/23732669/5c80ebea-04af-11e7-8023-f544be6610a6.JPG)
I hope _IntelliJ IDEA_ would give me a hint for `String`, but I still need me to write it myself. Did I miss something?

**Comment by framlog** [comment]
really? It did show me a list of hints after I typed `Str`. Perhaps you should check your settings of this plugin.

**Comment by NanXiao** [comment]
@framlg: "settings of this plugin.", could you elaborate this? Thanks very much!

**Comment by mkaput** [comment]
You have created a project in a directory which is **not** a Cargo project (you have two separate ones inside instead). Our requirement for almost any feature is that IntelliJ project root (~/rust in your situation) is also Cargo project/workspace root (it has Cargo.toml). Thanks to this we can firmly invoke cargo to get all information we need. 

tl;dr make separate projects inside each: hello_cargo and guessing_game :-)

**Comment by NanXiao** [comment]
@mkaput: Yes, you are definitely right! Thanks very much!
